jacques/kcqe.py

```diff
@@ -1,25 +1,39 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..')) 
 import numpy as np
 import tensorflow as tf
+from jacques.jacques import jacques
```

Comment:
again, not totally sure if this is correct
```suggestion
from .jacques import jacques
```